kiwi/system/setup.py

```diff
@@ -115,14 +115,24 @@ def import_repositories_marked_as_imageinclude(self):
         imageinclude attribute should be permanently added to
         the image repository configuration
         """
+        has_imageinclude = False
         repository_sections = self.xml_state.get_repository_sections()
+        for xml_repo in repository_sections:
+            if xml_repo.get_imageinclude():
+                has_imageinclude = True
+                break
+
+        if not has_imageinclude:
+            return
```

Comment:
Tip: This all can be written as one-liner:
```python
if not bool(next(xr for xr in self.xml_state.get_repository_sections() if xr.get_imageinclude())):
    return
```

But in general I am not fan of returning in the middle of the method. That means it is double-checking something inside that can be checked outside. I would rather check if this `has_imageinclude` and _only then_ would call this whole method _without_ double-checking things, assuming that the repos has included image.